xcp_d/workflow/postprocessing.py

```diff
@@ -216,6 +216,65 @@ def fwhm2sigma(fwhm):
     return fwhm / np.sqrt(8 * np.log(2))
 
 
+def init_pre_smoothing(mem_gb,
+                       presmoothing,
+                       omp_nthreads,
+                       cifti=False,
+                       name="presmoothing"):
+    """Smooth input BOLD before postprocessing."""
+    workflow = Workflow(name=name)
+    inputnode = pe.Node(niu.IdentityInterface(fields=['bold_file']),
+                        name='inputnode')
+    outputnode = pe.Node(niu.IdentityInterface(fields=['presmoothed_bold']),
+                         name='outputnode')
+
+    sigma_lx = fwhm2sigma(presmoothing)  # Turn specified FWHM (Full-Width at Half Maximum)
+    # to standard deviation.
+    if cifti:  # For ciftis
+        workflow.__desc__ = f""" \
+The preprocessed BOLD input was pre-smoothed using Connectome Workbench with a
+gaussian kernel size of {str(presmoothing)} mm (FWHM).
```

Comment:
```suggestion
Gaussian kernel size of {presmoothing} mm (FWHM).
```

I don't think you need to convert this to a string. The f-string formatter should handle it automatically.